{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/context_processors.py

```diff
@@ -0,0 +1,9 @@
+import urllib
+
+from django.conf import settings
+
+def global_vars(request):
+    return {
+        'BASE_URL': getattr(settings, 'BASE_URL', ""),
+        'DEBUG': getattr(settings, 'DEBUG', ""),
```

Comment:
DEBUG should be provided by Djangos builtin context ``debug`` context processor. Which is already enabled: https://github.com/torchbox/wagtail-cookiecutter/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.repo_name%7D%7D/settings/base.py#L79

``BASE_URL`` is only used for emails, you should use ``request.site.root_url`` instead.